README.md

```diff
@@ -279,13 +279,13 @@ Discord allows Bot-accounts to share load across sessions by limiting them to a
 Sharding will limit the amount of Guilds/Channels/Users visible to the JDA session so it is recommended to have some kind of elevated management to
 access information of other shards.
 
-To use sharding in JDA you will need to use `JDABuilder.useSharding(int shardId, int shardTotal)`. The **shardId** is 0-based which means the first shard
+To use sharding in JDA you will need to use `JDABuilder#useSharding(int shardId, int shardTotal)`. The **shardId** is 0-based which means the first shard
 has the ID 0. The **shardTotal** is the total amount of shards (not 0-based) which can be seen similar to the length of an array, the last shard has the ID of
 `shardTotal - 1`.
 
 The [`SessionController`](https://ci.dv8tion.net/job/JDA/javadoc/net/dv8tion/jda/api/utils/SessionController.html) is a tool of the JDABuilder
 that allows to control state and behaviour between shards (sessions). When using multiple builders to build shards you have to create one instance
-of this controller and add the same instance to each builder: `builder.setSessionController(controller)`
+of this controller and add the same instance to each builder: `JDABuilder#setSessionController(controller)`
```

Comment:
setSessionController isn't specific to JDABuilder so this change is not useful

Reply:
It's not? I changed it to `JDABuilder#` because earlier in that section is says that `SessionController` is a tool on the JDABuilder class.

Reply:
We also never use this notation (Type#method) in the documentation. It's also part of the shard manager.

Reply:
Isn't that proper Java documentation notation though? Considering it's part of the Shard Manager as well I'll revert it

Reply:
Not everyone is familiar with this notation

Reply:
But it is the standard, right?

Reply:
No there is no standard